MapleServer2/Servers/Game/FieldManager.cs

```diff
@@ -612,5 +613,15 @@ private Task StartSpawnTimer(IFieldObject<MobSpawn> mobSpawn)
                 }
             });
         }
+
+        public int Pin()
+        {
+            return ++Count;
+        }
+
+        public int Release()
+        {
+            return --Count;
+        }
```

Comment:
Unsure about the naming of this attribute and functions, can't think of something better! Any suggestions?